app/questionnaire/router.py

```diff
@@ -109,6 +109,10 @@ def _get_allowable_path(self, routing_path):
         """
         allowable_path = []
         for location in routing_path:
+            allowable_path.append(location)
+            if location not in self._completed_items.block_locations:
+                return allowable_path
+        return allowable_path
             if location in self._completed_store.block_locations:
                 allowable_path.append(location)
```

Comment:
Multiline suggestions don't really work 😄

Reply:
Already suggested by @iwootten 😛

Reply:
I was 👉 at it inserting Ian's suggestion instead of replacing with the older commit.

Reply:
Oh, yeah, that didn't work so well 😕